test/performance/odp_ipsec.c

```diff
@@ -25,7 +25,7 @@
 /** @def POOL_NUM_PKT
  * Number of packets in the pool
  */
-#define POOL_NUM_PKT  4096
+#define POOL_NUM_PKT  8192
```

Comment:
This POOL_NUM_PKT change does not appear to have anything to do with the vector support and the rest of the patch, so I think it should be taken out of this patch for clarity, and made through a separate patch if there is a reason for it.

Otherwise this patch looks ok. Reviewed-by: Janne Peltonen <janne.peltonen@nokia.com>